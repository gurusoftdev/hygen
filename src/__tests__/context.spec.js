const context = require('../context')
describe('context', () => {
  it('should populate with capitalized keys', () => {
    expect(context({ name: 'foobar' })).toMatchSnapshot()
  })
})
