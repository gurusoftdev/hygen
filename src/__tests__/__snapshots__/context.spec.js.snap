// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`context should populate with capitalized keys 1`] = `
Object {
  "Name": "Foobar",
  "h": Object {
    "capitalize": [Function],
    "inflection": Object {
      "_apply_rules": [Function],
      "camelize": [Function],
      "capitalize": [Function],
      "classify": [Function],
      "dasherize": [Function],
      "demodulize": [Function],
      "foreign_key": [Function],
      "humanize": [Function],
      "indexOf": [Function],
      "inflect": [Function],
      "ordinalize": [Function],
      "pluralize": [Function],
      "singularize": [Function],
      "tableize": [Function],
      "titleize": [Function],
      "transform": [Function],
      "undasherize": [Function],
      "underscore": [Function],
      "version": "1.12.0",
    },
  },
  "name": "foobar",
}
`;
